website/addons/dataverse/static/dataverseNodeConfig.js

```diff
@@ -286,36 +341,105 @@ ViewModel.prototype.getDatasets = function() {
     });
 };
 
-ViewModel.prototype.authorizeNode = function() {
+/** Send POST request to authorize Dataverse */
+ViewModel.prototype.sendAuth = function() {
     var self = this;
-    return osfHelpers.putJSON(
-        self.urls().importAuth, {}
-    ).done(function(response) {
-        self.updateFromData(response.result);
-        self.changeMessage(self.messages.importAuthSuccess, 'text-success', 3000);
-    }).fail(function(xhr, status, error) {
-        self.changeMessage(self.messages.importAuthError, 'text-danger');
-        Raven.captureMessage('Could not import Dataverse node auth', {
-            url: self.urls().importAuth,
-            textStatus: status,
+    var url = self.urls().create;
+    return $osf.postJSON(
+        url,
+        ko.toJS({
+            host: self.host,
+            api_token: self.apiToken
+        })
+    ).done(function() {
+        self.clearModal();
+        modal.modal('hide');
+        self.importAuth();
+    }).fail(function(xhr, textStatus, error) {
+        var errorMessage = (xhr.status === 401) ? self.messages.authInvalid : self.messages.authError;
+        self.changeMessage(errorMessage, 'text-danger');
```

Comment:
Doesn't necessarily need to go in this PR, but it'd be great if this ViewModel used the `ChangeMessageMixin` from `changeMessage.js`.